apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -44,113 +44,162 @@ export function InstructorAssessmentSettings({
-              <tbody>
-                <tr>
-                  <th scope="row">Title</th>
-                  <td>${resLocals.assessment.title}</td>
-                </tr>
-                <tr>
-                  <th scope="row">Type</th>
-                  <td>${resLocals.assessment.type}</td>
-                </tr>
-                <tr>
-                  <th scope="row">Set</th>
-                  <td>
-                    ${resLocals.assessment_set.name}
-                    <span class="text-muted">(${resLocals.assessment_set.abbreviation})</span>
-                  </td>
-                </tr>
-                <tr>
-                  <th scope="row">Number</th>
-                  <td>${resLocals.assessment.number}</td>
-                </tr>
-                <tr>
-                  <th scope="row">Module</th>
-                  <td>
-                    ${resLocals.assessment_module
-                      ? html`
-                          ${resLocals.assessment_module.heading}
-                          <span class="text-muted"> (${resLocals.assessment_module.name}) </span>
-                        `
-                      : html` &mdash; `}
-                  </td>
-                </tr>
-                <tr>
-                  <th scope="row">AID</th>
-                  <td>
-                    <span class="pr-2">${resLocals.assessment.tid}</span>
-                    ${resLocals.authz_data.has_course_permission_edit &&
+            <div class="card-body">
+              <form>
+                <div class="form-group">
+                  <label for="title">Title</label>
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="title"
+                    name="title"
+                    value="${resLocals.assessment.title}"
+                    disabled
+                  />
+                  <small class="form-text text-muted"> The title of the assessment. </small>
+                </div>
+                <div class="form-group">
+                  <label for="type">Type</label>
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="type"
+                    name="type"
+                    value="${resLocals.assessment.type}"
+                    disabled
+                  />
+                  <small class="form-text text-muted">
+                    The type of the assessment. This can be either Homework or Exam.
+                  </small>
+                </div>
+                <div class="form-group">
+                  <label for="set">Set</label>
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="set"
+                    name="set"
+                    value="${resLocals.assessment_set.name} (${resLocals.assessment_set
+                      .abbreviation})"
+                    disabled
+                  />
```

Comment:
This may be a good use case for [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) to simplify input based on existing sets. It would allow any set to be used, but would suggest the current sets as recommended options.

Reply:
This PR doesn't actually implement editing, but this is something to consider for the future PR where we build that out.